### PR TITLE
Adding test cases for manage_dbpatches roles

### PR DESCRIPTION
**Pull request by vibhorkum**
Adding test cases for manage_dbpatches roles

### CODE REVIEW

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1
```
Comment (jt-edb) [inline tests/docker/Dockerfile:1]: Please remove this file, we don't need it.

File: tests/tests/test_manage_dbpatches.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        query = r"\du"
+        cmd = host.run('psql -At -h %s -c "%s" postgres' % (socket_dir, query))
+        result = cmd.stdout.strip().replace('\nProfile', "").split('\n')
+        efm_index = [idx for idx, role in enumerate(result) if 'efm' in role][0]
+
+    assert 'pg_read_all_settings' in result[efm_index], \
+        "EFM role is not a pg_read_all_settings"
+
+def test_manage_dbpatches_redhat():
+    if not get_os().startswith('centos') and not get_os().startswith('rocky'):
```
Comment (jt-edb) [inline tests/tests/test_manage_dbpatches.py:67]: please import `os_family` and use it this way:
```python
   if os_family() != 'RedHat':
```

File: tests/tests/test_manage_dbpatches.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        'mailx',
+        'edb-efm44'
+    ]
+
+    for package in packages:
+        assert host.package(package).is_installed, \
+            "Package %s not installed" % packages
+
+def test_manage_dbpatches_debian():
+    if not (get_os().startswith('debian') or get_os().startswith('ubuntu')):
```
Comment (jt-edb) [inline tests/tests/test_manage_dbpatches.py:82]: Same comment about `os_family`

File: tests/config.yml [hunk 4]
```diff
@@ -297,3 +297,20 @@ cases:
...
+    pg_type:
+    - PG
+    - EPAS
+    os:
+    - centos7
+    - oraclelinux7
+    - rocky8
+    - debian9
+    - debian10
+    - ubuntu20
```
Comment (jt-edb) [inline tests/config.yml:316]: Could be interesting to test it against `oraclelinux7`.

File: tests/tests/test_manage_dbpatches.py (hunk 3 above)
Comment (vibhorkumar123) [reply tests/tests/test_manage_dbpatches.py]: fixed

File: tests/config.yml (hunk 4 above)
Comment (vibhorkumar123) [reply tests/config.yml:316]: fixed

File: tests/tests/test_manage_dbpatches.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+from conftest import (
+    load_ansible_vars,
+    get_hosts,
+    get_os,
+    get_pg_type,
+    get_primary,
+    get_standbys,
+    get_pg_unix_socket_dir,
+    get_pg_cluster_nodes
+)
```
Comment (jt-edb) [inline tests/tests/test_manage_dbpatches.py:13]: I think you have to import `os_family`

Comment (vibhorkumar123) [reply tests/tests/test_manage_dbpatches.py:14]: Fixed

**Pull request merged**